docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
I already removed my changes to the makefile and am instead calling the Go command directly in the travis script.